build_tools/docker/cmake/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# An image for building IREE using CMake.
+
+# Build using:
+# docker build --tag gcr.io/iree-oss/cmake build_tools/docker/cmake/
+
+# Set up the image and working directory.
+FROM ubuntu:18.04
+WORKDIR /usr/src/iree/
+
+RUN apt-get update
+
+# Update cmake to v3.13+, which is ahead of apt-get's version (3.10.2).
+# Install dependencies, including an old version of cmake to bootstrap.
+RUN apt-get install -y clang cmake libssl-dev wget
+ENV CMAKE_VERSION 3.13.5
+RUN mkdir ./cmake_install \
+  && cd cmake_install \
+  && wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.tar.gz \
+  && tar -xzvf cmake-${CMAKE_VERSION}.tar.gz \
+  && cd cmake-${CMAKE_VERSION}/ \
+  && cmake . \
+  && make \
+  && make install
+
+# Install dependencies.
+RUN apt-get install -y git
```

Comment:
run these (as well as the pip ones below) as all the same RUN - each RUN creates a new layer

Reply:
makes sense. I had things split up for caching purposes while I was iterating on things :)